src/cascadia/TerminalApp/TerminalSettingsSerializationHelper.h

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+#pragma once
+
+#include "JsonUtilsNew.h"
+#include "../TerminalSettings/terminalsettings.h"

```

Comment:
**absolutely not**

you're supposed to be including the **PROJECTED** header, because you're using TerminalSettings from another dll